<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StuDialog</class>
 <widget class="QDialog" name="StuDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>657</width>
    <height>434</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>657</width>
    <height>434</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>657</width>
    <height>434</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>学生成绩管理系统</string>
  </property>
  <widget class="QComboBox" name="value_comboBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>ID</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>score</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="cond_comboBox">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>20</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>升序</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>降序</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="sort_button">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>20</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>排序</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>401</width>
     <height>331</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="id_edit">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>90</y>
     <width>110</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="name_edit">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>160</y>
     <width>110</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="score_edit">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>230</y>
     <width>110</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="stu_id">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>90</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>学生学号</string>
   </property>
  </widget>
  <widget class="QLabel" name="stu_name">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>160</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>学生姓名</string>
   </property>
  </widget>
  <widget class="QLabel" name="stu_score">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>230</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>学生成绩</string>
   </property>
  </widget>
  <widget class="QPushButton" name="insert_button_2">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>310</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>插入</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delete_button">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>310</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="QPushButton" name="update_button">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>370</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>修改</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
